keyless/pepper/README.md

```diff
@@ -75,58 +113,3 @@ The script arguments are:
 - `<PEPPER_SERVICE_URL>`: The host and port of the Pepper Service, e.g., `http://localhost:8000`.
 - `<GOOGLE_PROJECT_ID>`: The ID of your GCP project.
 - `<FIRESTORE_DATABASE_ID>`: The ID of your Firestore database.
-
-Note: the client is an interactive console program, so you will need to follow the instructions
-to manually complete a session with the Pepper Service.
-
-### Frontend Developer Guide
-
-If you are a frontend developer, you should use the Pepper Client example as a reference to
-understand how to interact with the Pepper Service API. Your frontend application should
-implement the same user flows and API calls in the example.
-
-## Additional Testing
```

Comment:
Note: I've removed this from the README. All of the instructions below will be implemented as a unit test (in the next PR), so we no longer need this manual test case in the README 😄